jasmine-standalone-3.6.0 2/src/bowling.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+class Bowling {
+  constructor(){
+    this._rolls = [];
+  }
+
+  roll(pins){
+    this._rolls.push(pins)
+  }
+
+  reset(){
+    this._rolls = [];
+  }
+
+    get score(){
```

Comment:
As it is, this method doesn't quite work (needs to be one word). Adjusting to 'getScore' works ok :)